src/fontra/views/editor/visualization-layer-definitions.js

```diff
@@ -449,6 +458,13 @@ registerVisualizationLayerDefinition({
         context.globalAlpha = backgroundImage.color.alpha;
       }
       context.drawImage(image, 0, 0, image.width, image.height);
+
+      if (backgroundImageSelection) {
```

Comment:
This approach will transform the stroke as well. So if the image is small, the stroke will be thinner. Etc.

So, take this out of the `withSavedState()` block, set up the four corner points, and transform those, using `affine.transformPointObject()`, then use the resulting points to draw a polygon. We may need a helper function `drawPolygon(context, poinsts)`.

Additionally I suggest to use `context.lineJoin = "round";`

Reply:
I changed it as suggested, but instead of `drawPolygon` I named it `strokePolygon` to be in sync with other names like: `strokeCircle`, `strokeLineDashed`, `strokeLine`. I very much hope, this is in your interest as well.